baseband/guppi/frame.py

```diff
@@ -0,0 +1,83 @@
+# Licensed under the GPLv3 - see LICENSE
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..dada.frame import DADAFrame
+from .header import GUPPIHeader
+from .payload import GUPPIPayload
+
+
+__all__ = ['GUPPIFrame']
+
+
+class GUPPIFrame(DADAFrame):
```

Comment:
Now that we've made this less of a hack, I would suggest just inheriting from `VLBIFrameBase` - the only difference is that you then duplicate `DADAFrame.fromfile`, but I think that is OK. (Actually, the tiny difference between `fromdata` here and in `DADAFrame` suggests that perhaps `fromdata` should always allow just passing in a header to get `.bps`, etc., from. But that would be a different PR...)

Reply:
Done.